From dbebba11c3b80549ef80bd9005abe0c78a0dfdfe Mon Sep 17 00:00:00 2001
From: Daan De Meyer <daan.j.demeyer@gmail.com>
Date: Sat, 21 Dec 2024 17:01:27 +0100
Subject: [PATCH] units: Order systemd-oomd after systemd-sysusers

systemd-sysusers might create the systemd-oom system user that
systemd-oomd runs under so let's order systemd-oomd after
systemd-sysusers.
---
 units/systemd-oomd.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-oomd.service.in b/units/systemd-oomd.service.in
index 670d5e6140814..6838ddccef46a 100644
--- a/units/systemd-oomd.service.in
+++ b/units/systemd-oomd.service.in
@@ -20,7 +20,7 @@ ConditionPathExists=/proc/pressure/cpu
 ConditionPathExists=/proc/pressure/io
 ConditionPathExists=/proc/pressure/memory
 Requires=systemd-oomd.socket
-After=systemd-oomd.socket
+After=systemd-oomd.socket systemd-sysusers.service
 
 [Service]
 AmbientCapabilities=CAP_KILL CAP_DAC_OVERRIDE
